src/Security/Service/SecurityService.php

```diff
@@ -75,23 +72,16 @@ public function checkAuthToken(string $token): bool
     }
 
     /**
-     * @throws NotAuthorizedException
+     * @throws UserNotFoundException
      */
     public function getCurrentUser(): UserInterface
     {
-        $entry = $this->tmpStoreResolver->get($this->tokenService->getCurrentToken());
-
-        if($entry === null || !is_array($entry->getData()) || !isset($entry->getData()['username'])) {
-            throw new NotAuthorizedException();
+        $pimcoreUser = $this->authenticationResolver->authenticateSession();
+        if (!$pimcoreUser instanceof User) {
+            throw new UserNotFoundException('No pimcore user found');
```

Comment:
Should we really switch to a user not found exception instead of a NotAuthorized?

Reply:
Since the Token check was removed, we do not really check for authentication here. We just grab the Authenticated user. So that's my i would leave this Exception.

Reply:
But if you try to grab the user the only reason why you would not get it, is because you are not authorized right? :)

Reply:
IMHO then you should not be able to enter the Controller at all. And this is done by the Access Control. 

And since the method says getCurrentUser. I think it makes more sense to throw a UserNotFoundException.

Reply:
Agreed! :)